Makefile

```diff
@@ -15,13 +15,17 @@ LUMA_SYSMODULE_OUT  := luma/sysmodules
 
 FRIENDS_TITLE_ID	:= 0004013000003202
 ACT_TITLE_ID	:= 0004013000003802
+HTTP_TITLE_ID    := 0004013000002902
+SOCKET_TITLE_ID    := 0004013000002E02
 SSL_TITLE_ID    := 0004013000002F02
 MIIVERSE_ID_JPN     := 000400300000BC02
 MIIVERSE_ID_USA     := 000400300000BD02
 MIIVERSE_ID_EUR     := 000400300000BE02
```

Comment:
Mixed tabs and spaces, unusable /s

Reply:
tbf that isn't my code, it was already like that